charts/orchestrator/templates/_cr_lifecycle_helper.tpl

```diff
@@ -56,73 +52,74 @@ roleRef:
   kind: ClusterRole
   name: {{ $releaseNameKind }}
   apiGroup: rbac.authorization.k8s.io
-    {{- if .isEnabled }}
 ---
 apiVersion: batch/v1
-kind: Job
+kind: CronJob
 metadata:
-  name: {{ trunc -57 (printf "%s-%s" $releaseNameKind (.release.IsInstall | ternary "install" "upgrade" ) ) }} # Fixes https://github.com/parodos-dev/orchestrator-helm-chart/issues/160
+  name: {{ trunc -52 (printf "%s-reconcile" $releaseNameKind | trimPrefix "-" | trimPrefix "_" ) }} # Fixes https://github.com/parodos-dev/orchestrator-helm-chart/issues/160
   # job name is used in the spec.template.metadata.labels, and labels cannot be more than 63 characters https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
   namespace: {{ .release.Namespace }}
-  annotations:
-    "helm.sh/hook": post-install,post-upgrade,post-rollback
-    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
-    "helm.sh/hook-weight": "1"
+  labels:
+    "orchestrator.rhdh.redhat.com/reconciles": {{ $resourceAPIGroup }}
 spec:
-  template:
-    metadata:
-      name: {{ printf "%s-upgrade" $releaseNameKind }}
+  schedule: '* * * * *' #run every minute
```

Comment:
Does it mean this CronJob remains running on the cluster after the helm install is completed?

Reply:
Yes. It's not what a helm install should do, but it is expected in the operator.

Reply:
I'm going to merge this PR, but I feel that the limitation of the helm-based operator are slowly pushing us toward a pretty complex implementation which makes me start wondering what should be the breaking point for switching to GoLang-based operator.